apps/utils/locust_tests/v1/regression_suite.py

```diff
@@ -77,17 +78,17 @@ class RegressionV1User(BFDUserBase):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.bene_ids = master_bene_ids.copy()
-        self.contract_data = master_contract_data.copy()
-        self.hashed_mbis = master_hashed_mbis.copy()
+        self.bene_ids = itertools.cycle(master_bene_ids.copy())
+        self.contract_data = itertools.cycle(master_contract_data.copy())
+        self.hashed_mbis = itertools.cycle(master_hashed_mbis.copy())
 
     @tag("coverage", "coverage_test_id_count")
     @task
     def coverage_test_id_count(self):
         """Coverage search by ID, Paginated"""
         self.run_task_by_parameters(
             base_path="/v1/fhir/Coverage",
-            params={"beneficiary": self.bene_ids.pop(), "_count": "10"},
+            params={"beneficiary": next(self.bene_ids), "_count": "10"},
```

Comment:
[`next(...)`](https://docs.python.org/3/library/functions.html?highlight=next#next) simply returns the next element from the iterator created by `itertools.cycle(...)` above.